.travis.yml

```diff
@@ -6,6 +6,9 @@ addons:
 jdk:
 - oraclejdk8
 install: true
+before_script:
```

Comment:
This is necessary to allow the new integration tests added in this PR to run on Travis.  The integration tests spin up a new EDT, thus there needs to be a virtual display in order to avoid a `HeadlessException`.